internal/devconfig/file.go

```diff
@@ -33,10 +33,11 @@ const (
 	tsonFormat
 )
 
-// configFile defines a devbox environment as JSON.
-type configFile struct {
+// ConfigFile defines a devbox environment as JSON.
+type ConfigFile struct {
 	Name        string `json:"name,omitempty"`
 	Description string `json:"description,omitempty"`
+	Version     string `json:"version"`
```

Comment:
Can we make this a separate change for now? It would be nice if `devbox.json` didn't require a version field.